readLog/src/main/java/org/corfudb/readLog/LogEntryExtended.java

```diff
@@ -0,0 +1,22 @@
+package org.corfudb.readLog;
+
+import lombok.Getter;
+import org.corfudb.format.Types.LogEntry;
+
+/**
+ * Created by kjames88 on 3/2/17.
+ */
+
+public class LogEntryExtended {
```

Comment:
This class allows more than one value to be returned (to avoid using a tuple (un-java-like?))  Adding a metadata would again result in more than one return value.